newIDE/app/src/UI/List.js

```diff
@@ -193,6 +193,7 @@ export class ListItem extends React.Component<ListItemProps, ListItemState> {
           <MUIListItemText
             primary={props.primaryText}
             secondary={props.secondaryText}
+            style={{ margin: '0 auto' }}
```

Comment:
Creating a styles object outside of the component, and point to it: `style={styles.listItemText}`.
This avoids a new object to be created at each render, reducing pressure on the GC.

Reply:
I will change it as advised and will try to adopt the best practices in the future.